src/widgets/DataTable/DataTable.js

```diff
@@ -6,7 +6,7 @@
 
 import PropTypes from 'prop-types';
 import React, { useMemo, useRef, useCallback } from 'react';
-import { StyleSheet, VirtualizedList, VirtualizedListPropTypes } from 'react-native';
+import { StyleSheet, VirtualizedList, VirtualizedListPropTypes, Keyboard } from 'react-native';
```

Comment:
have these changes snuck in from another branch?
I remember reviewing a PR with this in it last week

Reply:
hmm yeah https://github.com/openmsupply/mobile/pull/1291/files -- I'm not sure why that is coming up in the diff. I will merge this after the other PR's and ensure I have based correctly